src/shared/test_util/tbots_gtest_main.cpp

```diff
@@ -26,6 +26,7 @@ int main(int argc, char **argv)
         if (TbotsGtestMain::enable_visualizer)
         {
             TbotsGtestMain::stop_ai_on_start = args->getStopAiOnStart()->value();
+            fedisableexcept(FE_INVALID | FE_OVERFLOW);
```

Comment:
Just curious what does this do?

Reply:
This is to fix a floating-point error when running simulated tests with the visualizer (might be a problem exclusive to WSL or Ubuntu 20). @jonathanlew can comment more on it

Reply:
there's a sigfe in the qt library when running wsl, so this disables sigfe only when the visualizer is on

Reply:
Um, we should leave a _very_ detailed comment on why this is needed.